### ISSUE TITLE
Add the Rating column to the Playlist view as well (and: Multiselect & Rate)

### ISSUE THREAD

**Issue by WolfgangDpunkt** [navidrome/navidrome#1473]
It would be great if you could add the "Rating" column (5-stars rating) to the (smart) playlist view, just like it is already possible with the Songs view.

An extra cool feature would be, to be than able to select multiple songs and give them the same rating at the same time (e.g. give all selected songs of an playlist 3 stars).

This would allow to deal with the problem that the ratings stored in the tags by other applications (like MusicBee) are not supported  (https://github.com/navidrome/navidrome/issues/1301).
One could then e.g. prepare in other apps playlists with all the 3-star-songs and assign them in Navidrome than all the 3-star-rating in the Subsonic/Navidrome-format using the multiple selection and rating feature.

This would allow music enthusiasts to move their carefully rated music collections more easily to Navidrome.

Thank you for considering this feature. :-)

**Comment by deluan** [comment]
> It would be great if you could add the "Rating" column (5-stars rating) to the (smart) playlist view, just like it is already possible with the Songs view.

Yeah, I tried this in the past (also adding the "Favourite" button), but there are some challenges to implement this for playlists. I'll give it another try.

> An extra cool feature would be, to be than able to select multiple songs and give them the same rating at the same time (e.g. give all selected songs of an playlist 3 stars).
> 
> This would allow to deal with the problem that the ratings stored in the tags by other applications (like MusicBee) are not supported (#1301). One could then e.g. prepare in other apps playlists with all the 3-star-songs and assign them in Navidrome than all the 3-star-rating in the Subsonic/Navidrome-format using the multiple selection and rating feature.
> 
> This would allow music enthusiasts to move their carefully rated music collections more easily to Navidrome.

That's a great idea! Once I solve the issue above, I can take a look at this

**Comment by certuna** [comment]
I've created a separate issue for the multiselect/rate suggestion: https://github.com/navidrome/navidrome/issues/1479

**Comment by bcsteeve** [comment]
No luck on the ratings view in Playlists?  It is very useful for folks like me that have general "favs" lists based on star ratings.   In my player, I'll maybe bore of a song and "downgrade it" by lowering the stars.  Then, ideally, I would go to the playlist and sort by rating and trim off the ones that no longer make the cut.

As it is, I have to basically delete the playlist and make a new one.  Not a big deal, but enough of a pain that I came looking for how to show ratings in playlist view! 

Cheers.

**Comment by deluan** [comment]
> It is very useful for folks like me that have general "favs" lists based on star ratings. In my player, I'll maybe bore of a song and "downgrade it" by lowering the stars. Then, ideally, I would go to the playlist and sort by rating and trim off the ones that no longer make the cut.

You can use a Smart Playlist (#1417) for that use case:

```json
{
  "all": [
    {"gt": {"rating": 3}}
  ],
  "sort": "rating",
  "order": "desc",
  "limit": 500
}
```

It will update automatically when you change ratings of any song.

**Comment by bcsteeve** [comment]
> You can use a Smart Playlist (#1417) for that use case:

Awesome!  I had no idea that was a thing. Thanks.

**Comment by graham2071** [comment]
I am also interested in this feature. I use Ratings a lot, and smart playlists based on ratings. When I want to change the rating of a song from the playlist, I need to open the album view.

**Comment by matias-autio** [comment]
Is this still in the works? I would love this feature as well :)
